layouts/partials/blocks/text-2.html

```diff
@@ -10,7 +10,7 @@
         >
           {{ .title }}
         </h2>
-        <div class="max-w-3xl text-xl text-white lg:pl-20 lg:text-2xl">
+        <div class="article-content  max-w-3xl text-lg text-white lg:pl-20">
```

Comment:
Run format to fix this spacing.

Reply:
resolved with https://github.com/spotlightpa/poor-richard/pull/338/commits/bfc94b11ea5d19ef87a6e4a16ac21d040eb84f5f